tests/doxygen/message-format.dox

```diff
@@ -0,0 +1,199 @@
+ * command response. Typically the host will increment the token with each
+ * request.
+ * @param Command The command ID.
+ * @param Return_Code The \ref ReturnCode of the command.
+ * @param Status The status bitfield.
+ * @param Length The length of the data included in the command. The valid
+ * range is 0 - 579 bytes.
+ * @param Payload The payload data associated with the command. See each
+ * command type below for the specific format of the payload data.
+ * @param EOF The end of frame identifier: \ref END_OF_MESSAGE_ID
+ * @param Padding Extra padding, the Host should set this to 0. Padding should
+ * not cause the total frame to exceed \ref USB_READ_BUFFER_SIZE.
+ *
+ * @section frame_commands Commands
+ *
+ * @subsection frame_cmd_echo Echo
+ * Echo data back from the device. This serves as a basic test that Host to
+ * Device communications are working correctly.
+ *
+ * @subsubsection frame_cmd_echo_req Request Payload
+ *
+ * <pre>
+ *    0                   1                   2                   3
+ *    0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+ *   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ *   \                         Data (if any)                         \
+ *   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ * </pre>
+ *
+ * @param data 0 or more bytes of data.
+ *
+ * @subsubsection frame_cmd_echo_res Response Payload
+ *
+ * <pre>
+ *    0                   1                   2                   3
+ *    0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+ *   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ *   \                         Data (if any)                         \
+ *   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ * </pre>
+ *
+ * @param data 0 or more bytes of data, to match what was provided in the
+ * request.
+ * @returns \ref RC_OK.
+ *
+ * @subsection frame_cmd_getbreak Get Break Time
+ *
+ * Gets the current break time for outgoing DMX512 / RDM frames.
+ *
+ * @subsubsection frame_cmd_getbreak_req Request Payload
+ *
+ * The request contains no data.
+ *
+ * @subsubsection frame_cmd_getbreak_res Response Payload
+ *
+ * <pre>
+ *    0                   1
+ *    0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5
+ *   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ *   |         Break_time            |
+ *   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ * </pre>
+ *
+ * @param Break_time The current break time in microseconds.
+ * @returns \ref RC_OK.
+ *
+ * @subsection frame_cmd_setbreak Set Break Time
+ *
+ * Sets the break time for outgoing DMX512 / RDM frames.
+ *
+ * @subsubsection frame_cmd_setbreak_req Request Payload
+ *
+ * <pre>
+ *    0                   1
+ *    0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5
+ *   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ *   |         Break_time            |
+ *   +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+ * </pre>
+ *
+ * @param Break_time The new break time. See Transceiver_SetBreakTime().
```

Comment:
The name is automatically linked. Using @see puts it in a separate section which isn't what I want.